### DISCUSSION TITLE
[Remix] use a Remix action to log in with auth-helpers

### DISCUSSION THREAD

**Discussion by Kilian** [supabase/supabase#12210] [Feature Requests]
The Supabase auth-helpers for Remix want you to log in with client-side logic, which sets a cookie that the server can then read out.

The Remix way of handling login is with form submissions to a server side `action` that then performs the authentication and sets the appropriate cookies. It would be great if the auth-helper library could be updated to support this flow, or if documentation could be written for that flow to be supported.

**Comment by bwalsh1191** [comment]
having the same issues here. Already followed the egghead.io course too and still confused
